src/main/java/org/wildfly/installationmanager/HistoryResult.java

```diff
@@ -19,20 +19,23 @@
 package org.wildfly.installationmanager;
 
 import java.time.Instant;
+import java.util.List;
 import java.util.Objects;
 
 public class HistoryResult {
 
+    private final List<ManifestVersion> versions;
     private String hash;
     private Instant timestamp;
     private String type;
     private String description;
 
-    public HistoryResult(String hash, Instant timestamp, String type, String description) {
+    public HistoryResult(String hash, Instant timestamp, String type, String description, List<ManifestVersion> versions) {
```

Comment:
We should @deprecate the old constructor for removal and internally call this passing an empty List for the versions since this will be a breacking change